src/local/room_name.rs

```diff
@@ -61,20 +61,22 @@ impl fmt::Display for RoomName {
         let x_coord = self.x_coord();
         let y_coord = self.y_coord();
 
+        #[cfg(feature = "sim")]
         if self.packed == 0 {
             write!(f, "sim")?;
+            return Ok(());
+        }
```

Comment:
```suggestion
        if cfg!(feature = "sim") && self.packed == 0 {
            write!(f, "sim")?;
            return Ok(());
        }
```
As discussed elsewhere, this is free and improves checks.